src/overlays/actors/ovl_En_Mushi2/z_en_mushi2.h

```diff
@@ -10,39 +10,39 @@ typedef void (*EnMushi2ActionFunc)(struct EnMushi2*, PlayState*);
 #define ENMUSHI2_GET_3(thisx) ((thisx)->params & 3)
 
 typedef struct {
-    /* 0x00 */ s16 unk_00;
-    /* 0x02 */ s16 unk_02;
+    /* 0x0 */ s16 unk_00;
+    /* 0x2 */ s16 unk_02;
 } EnMushi2Struct;
```

Comment:
Should remove the `0x`s to be consistent, and add the size

Reply:
Usage does not suggest a normal `Vec2s`. Also I think I prefer `0x` for these, I'll change to this style if that is okay.

Reply:
As far as I know all structs should have a `0x`, it is the enums that are inconsistent with each other and with structs.